nbxplorer/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nbxplorer.fullname" . }}
+  labels:
+    {{- include "nbxplorer.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      {{- include "nbxplorer.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "nbxplorer.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "nbxplorer.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          args:
+            - "--datadir=/data"
+            - "--bind=0.0.0.0:32838"
+            - "--network=mainnet"          
+            - "--chains=btc"
+            - "--noauth"
+            - "--btcrpcurl=http://bitcoind.bitcoin.svc.cluster.local:8332/"
+            - "--btcnodeendpoint=bitcoind.bitcoin.svc.cluster.local:8333"            
```

Comment:
`btcnodeendpoint` and `btcrpcurl` should be configurable. Also why is one using `8333` and the other is using `8332`? Secure vs unsecure...